.github/workflows/periodic.yml

```diff
@@ -0,0 +1,46 @@
+name: Periodic Functional Tests
+
+on:
+  schedule:
+    - cron: '*/5 * * * *' # Run every 5 minutes
+
+jobs:
+  run_tests:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+      jobs:
+      setup_kind:
+      runs-on: ubuntu-latest
+      steps:
+    - name: Check if Docker is installed
+      run: |
+        if ! command -v docker &> /dev/null
+        then
+            echo "Docker is not installed. Installing Docker..."
+            sudo apt-get update
+            sudo apt-get install -y docker.io
+        else
+            echo "Docker is already installed."
+        fi
+    - name: Install KinD
+      run: |
+        curl -Lo kind https://kind.sigs.k8s.io/dl/v0.11.1/kind-linux-amd64
+        chmod +x kind
+        sudo mv kind /usr/local/bin
+    - name: Verify KinD installation
+      run: kind version        
+    - name: Run Functional Tests
+      run: 
+        ./test/kfp-functional-test/kfp-functional-test.sh
+    - name: Collect test results
```

Comment:
Although the step is present in the workflow, it requires some changes in the bash script to generate the artifacts.
* Path creation is done [here](https://github.com/kubeflow/pipelines/blob/master/scripts/deploy/github/test-pipeline.sh#L35)
* Artifact collection is done [here](https://github.com/kubeflow/pipelines/blob/master/scripts/deploy/github/test-pipeline.sh#L49)

Basically, what you can do is redirect [this execution](https://github.com/kubeflow/pipelines/blob/master/test/kfp-functional-test/kfp-functional-test.sh#L22) to a path that you create previously.